warehouse/models/staging/transit_database/stg_transit_database__county_geography.sql

```diff
@@ -0,0 +1,28 @@
+{{ config(materialized='table') }}
+
+WITH
+latest AS (
+    {{ get_latest_external_data(
+        external_table = source('airtable', 'california_transit__county_geography'),
+        order_by = 'dt DESC, time DESC'
+        ) }}
+),
+
+stg_transit_database__county_geography AS (
+    SELECT
+        county_geography_id AS key,
+        {{ trim_make_empty_string_null(column_name = "name") }},
+        fips,
+        msa,
+        caltrans_district,
+        caltrans_district_name,
+        unnested_rtpa AS rtpa,
```

Comment:
this was my bad -- can we update these to be like `rtpa_organization_key` to list what table it's a foreign key to? (would also need to update YAML with new name)